etc/org.opencastproject.userdirectory.ldap.cfg.template

```diff
@@ -50,6 +50,19 @@ org.opencastproject.userdirectory.ldap.roleattributes=
 #org.opencastproject.userdirectory.ldap.userDn=
 #org.opencastproject.userdirectory.ldap.password=
 
+## The attributes used to construct the user details.
+## If left empty, the user details wont be mapped.
+## The name config key is a comma-separated list of attributes that will be joined
+##   with a whitespace character in the order they appear.
+## Example:
+##  org.opencastproject.userdirectory.ldap.userattributes.name=givenName,sn
+##  org.opencastproject.userdirectory.ldap.userattributes.mail=mail
+## Result:
+##   name: John Doe
+##   mail: john.doe@example.org
+#org.opencastproject.userdirectory.ldap.userattributes.name=
+#org.opencastproject.userdirectory.ldap.userattributes.mail=
```

Comment:
Tried and works.


One question, can you add custom ldap fields? Because we have another attributes to identify if is a student or a employee. And also if belongs to the uni.

Anyways, works well.

Reply:
What do you want to achieve with the LDAP fields?

Reply:
Can you add custom fields? The current LDAP you can add roles from a custom parameter is set in LDAP, but not save it like is with the name or email.